src/tools/pub/dependencies.md

```diff
@@ -174,6 +173,20 @@ The ref can be anything that Git allows to [identify a commit][commit].
 
 [commit]: http://www.kernel.org/pub/software/scm/git/docs/user-manual.html#naming-commits
 
+Pub assumes that the package is in the root of the Git repository.  To specify a different
+location in the repo use the `path` argument:
+
+{% prettify yaml %}
+dependencies:
+  kittens:
+    git:
+      url: git://github.com/munificent/cats.git
+      ref: some-branch
```

Comment:
As suggested in https://github.com/dart-lang/site-www/pull/773#discussion_r180771723, remove `ref`.

Reply:
Done.